README.md

```diff
@@ -139,6 +139,25 @@ Together with the `uniqueItems` property (which is native to JSON Schema), compl
 | true           | true           | ordered set    |
 | false          | true           | set      |
 
+#### optionPrefix/Suffix
```

Comment:
just in case anyone is searching for documentation
```suggestion
#### optionalPrefix/optionalSuffix
```

Reply:
sure